include/bluetooth/bluetooth.h

```diff
@@ -443,6 +443,12 @@ struct bt_le_scan_param {
 	u16_t window;
 };
 
+/** Listener context for (LE) scanning. */
+struct bt_le_scan_listener {
+	bt_le_scan_cb_t *cb; /**< Callback to call on every scan packet. */
```

Comment:
Can we rename this cb to device_found?
For AE I might want to add another callback. stopped/timeout.

Reply:
Discussed offline, will change to `recv`